### DISCUSSION TITLE
Is Alarms subscription supported?

### DISCUSSION THREAD

**Discussion by mshekhar212** [eclipse-milo/milo#1225] [General]
Hi, I am trying to get notification whenever an Alarm is active from KEPServerEX. I tried dOPC AE Client it is getting active alarms from the same KEPServerEX but I am not able to make it work through Milo.
![image](https://github.com/eclipse/milo/assets/36514073/6b2247cc-3e34-4339-803d-406802c2eaf9)

My question is does milo support getting list of Alarms and then subscribing to any specific alarm (i.e getting notification for only subscribed alarms and not all alarms). If it is supported can you please suggest any example or way to do it. I have tried EventSubscriptionExample and ManagedSubscriptionEventExample but could not make it work.

Thank you

**Comment by kevinherron** [comment]
Yes, event subscriptions are supported. 

Are you sure Kepware supports OPC UA Alarms and Conditions? And that your test client isn't an OPC Classic (COM/DCOM) client?
